src/pio/project.rs

```diff
@@ -214,7 +214,7 @@ impl Builder {
         self
     }
 
-    pub fn enable_cargo(&mut self, cargo: cargo::Cargo) -> &mut Self {
+    pub fn enable_cargo(&mut self, cargo: cargo::CargoCmd) -> &mut Self {
```

Comment:
As per the above comment for "init" vs "new" you might have to alter this code slightly again.